content/en/docs/instrumentation/go/getting-started.md

```diff
@@ -4,160 +4,79 @@ weight: 10
 cSpell:ignore: chan codebases fscanf println stdouttrace strconv struct
 ---
 
-Welcome to the OpenTelemetry for Go getting started guide! This guide will walk
-you through the basic steps in installing, instrumenting with, configuring, and
-exporting data from OpenTelemetry. Before you get started, be sure to have Go
-1.16 or newer installed.
-
-Understanding how a system is functioning when it is failing or having issues is
-critical to resolving those issues. One strategy to understand this is with
-tracing. This guide shows how the OpenTelemetry Go project can be used to trace
-an example application. You will start with an application that computes
-Fibonacci numbers for users, and from there you will add instrumentation to
-produce tracing telemetry with OpenTelemetry Go.
-
-For reference, a complete example of the code you will build can be found
-[here](https://github.com/open-telemetry/opentelemetry-go/tree/main/example/fib).
-
-To start building the application, make a new directory named `fib` to house our
-Fibonacci project. Next, add the following to a new file named `fib.go` in that
-directory[^1].
-
-[^1]:
-    The `Fibonacci()` function intentionally produces invalid results for
-    sufficiently large values of `n`. This is addressed in
-    [Error handling](#error-handling).
+This page will show you how to get started with OpenTelemetry in Golang.
 
-```go
-package main
+You will learn how you can instrument a simple Go application automatically,
+in such a way that [traces][], [metrics][] and [logs][] are emitted to the
+console.
 
-// Fibonacci returns the n-th Fibonacci number.
-func Fibonacci(n uint) (uint64, error) {
-	if n <= 1 {
-		return uint64(n), nil
-	}
-
-	var n2, n1 uint64 = 0, 1
-	for i := uint(2); i < n; i++ {
-		n2, n1 = n1, n1+n2
-	}
+## Prerequisites
 
-	return n2 + n1, nil
-}
-```
+Ensure that you have the following installed locally:
 
-With your core logic added, you can now build your application around it. Add a
-new `app.go` file with the following application logic.
+- Golang JDK 1.18+
 
-```go
-package main
+## Example Application
 
-import (
-	"context"
-	"fmt"
-	"io"
-	"log"
-)
+The following example uses a basic
+[Gin Web Framework](https://gin-gonic.com/docs/introduction/). If you are
+not using Gin, that's OK — you can use OpenTelemetry Golang with other web
+frameworks as well, such as Goframe and Beego. 
```

Comment:
Can we simply use `net/http`? You can mention that it works for Gin, Gorilla/Mux, Echo etc.